src/components/organisms/Header/constants.tsx

```diff
@@ -19,16 +19,19 @@ export const PRODUCTION_LINKS = [
         text: 'Bridge',
         link: '/bridge',
         dataCy: 'bridge',
+        className: 'hidden desktop:flex',
     },
     {
         text: 'Points',
         link: '/points',
         dataCy: 'points',
+        className: 'flex laptop:hidden',
```

Comment:
Points is hidden on laptop and desktop?

Reply:
I see points goes inside the more dropdown on laptop but desktop it's not present.

Reply:
>Points is hidden on laptop and desktop?

Yes. On desktop the Points badge is used as a link instead

> I see points goes inside the more dropdown on laptop but desktop it's not present.

Yes. Ben updated the header that way

Reply:
ah. shouldn't then points badge be a link for all breakpoints.
As a user I was looking how to switch to the points page on desktop.
Also desktop had sufficient space on the header, so it was not really necessary to hide points and take the link to the points tag.

Reply:
i think it mightve been Aki-san who mentioned to swap the Points with the Stablecoin. we can leave the link button as is on breakpoints that have enough space but it gets cramped very quickly